kalite/distributed/tests/ajax_messages_test.py

```diff
@@ -0,0 +1,20 @@
+from selenium.common.exceptions import NoSuchElementException
+
+from kalite.testing.base import KALiteBrowserTestCase
+from kalite.testing.mixins import BrowserActionMixins, FacilityMixins
+
+
+class TestShowAjaxMessages(BrowserActionMixins, FacilityMixins, KALiteBrowserTestCase):
+
+    def test_ajax_messages(self):
+
+        # Navigate to video that is not downloaded
+
+        self.browse_to(
+            self.reverse("learn") + "khan/math/algebra/introduction-to-algebra/overview_hist_alg/origins-of-algebra/")
```

Comment:
Presumably if the video was downloaded the test would fail without actually testing whether the appropriate error message shows up.